packages/rocketchat-lib/server/methods/createPrivateGroup.coffee

```diff
@@ -1,5 +1,9 @@
 Meteor.methods
 	createPrivateGroup: (name, members) ->
+
+		check name, String
+		check members, [String]
```

Comment:
members is optional
If you check the function it calls RocketChat.createRoom, you'll see that members is not required, as it will be set to [] if none is passed.